python/fbprophet/forecaster.py

```diff
@@ -1474,3 +1524,58 @@ def copy(self, cutoff=None):
             DeprecationWarning,
         )
         return prophet_copy(m=self, cutoff=cutoff)
+
+    def make_holidays(self, year_list, country_list):
+        """Make dataframe of holidays for given years and countries
+
+        Parameters
+        ----------
+        cls: Prophet class
+        year_list: a list of years
+        country_list: a list of countries
+
+        Returns
+        -------
+        Dataframe with 'ds' and 'holiday', which can directly feed
+        to 'holidays' params in Prophet
+        """
+
+        if isinstance(country_list, str):
+            all_hdays = []
+            for year in year_list:
+                try:
+                    temp = getattr(hdays_part1, country_list)(years=year)
+                except AttributeError:
+                    try:
+                        temp = getattr(hdays_part2, country_list)(years=year)
+                    except AttributeError:
+                        raise AttributeError(
+                            "Holidays in {} are not currently supported!".format(country_list))
+                temp_df = pd.DataFrame(list(temp.items()),
+                                       columns=['ds', 'holiday'])
+                all_hdays.append(temp_df)
+
+        elif isinstance(country_list, list):
+            all_hdays = []
+            for country in country_list:
+                for year in year_list:
+                    try:
+                        temp = getattr(hdays_part1, country)(years=year)
+                    except AttributeError:
+                        try:
+                            temp = getattr(hdays_part2, country)(years=year)
+                        except AttributeError:
+                            raise AttributeError(
+                                "Holidays in {} are not currently supported!".format(country))
+                    temp_df = pd.DataFrame(list(temp.items()),
+                                           columns=['ds', 'holiday'])
+                    all_hdays.append(temp_df)
+        res = pd.concat(all_hdays, axis=0, ignore_index=True)
+
+        res = res.groupby(['ds']) \
+            .first() \
```

Comment:
Am I correct in understanding that this `.first()` means that we only allow one holiday per ds? That would not be necessary, and I think not what we want to do.